lib/src/virtual_directory.dart

```diff
@@ -114,7 +115,8 @@ class VirtualDirectory {
     _errorCallback = callback;
   }
 
-  Future _locateResource(String path, Iterator<String> segments) async {
+  Future<dynamic?> _locateResource(
```

Comment:
`dynamic` already nullable. This should probably be `Object?` though and we should explicitly cast if we need to invoke members?

Reply:
Done.